Resources/Maps/frontier.yml

```diff
@@ -4972,6 +4965,18 @@ entities:
     - pos: -58.5,0.5
       parent: 2173
       type: Transform
+- proto: AirlockFrontierCommandGlassLocked
+  entities:
+  - uid: 2384
+    components:
+    - pos: 35.5,14.5
+      parent: 2173
+      type: Transform
```

Comment:
Do we need the SR to be able to access the Sheriff's office? (The above specific door is the relevant one.)

Reply:
Might as well allow them so they can store contraband in that office if need be.
Also we can probably use another empty safe in some sec area, not sure where to place it

Reply:
Fair enough, will consider this in upcoming redesign of NFSD.